### DISCUSSION TITLE
motif footprinting interpretation

### DISCUSSION THREAD

**Discussion by Ping-lin14** [GreenleafLab/ArchR#688] [Questions / Documentation]
My question is, why is this graph an undulating curve instead of a smooth curve?
Just like the blue handwriting.
Does this have anything to do with bulk ATAC?
![InkedFootprints-Subtract-Bias_1_LI](https://user-images.githubusercontent.com/69965537/114304114-13d78d00-9b04-11eb-86c5-d8911cf631d9.jpg)

**Comment by rcorces** [comment]
I'm not sure I understand your question. The "undulating curve" is just noise in the data for the most part. You can adjust the `smoothWindow` to smooth out the curve.

**Reply by Ping-lin14** [reply]
I'm not sure my understanding is correct.
The position of transcription factor binding directly protects the position of DNA base, which makes transposition difficult.
It can be considered that the closer to TF binding site, the lower the accessibility of chromatin, so the restlessness gradually increased.

**Comment by Cailu086Lin** [comment]
How can we  modify the labeled text size (cluster name) or remove it in this plotFootprints?

Thanks

**Reply by rcorces** [reply]
Please refrain from asking secondary questions within a post.
No - you cannot modify this in ArchR. You are welcome to do so manually in adobe illustrator or use `plot = FALSE` and plot the footprints yourself however you would like.
